llvm/lib/Target/AIE/AIELegalizerHelper.cpp

```diff
@@ -153,17 +162,20 @@ bool AIELegalizerHelper::legalizeG_BUILD_VECTOR(LegalizerHelper &Helper,
 
   const AIEBaseInstrInfo *II = ST.getInstrInfo();
   MachineOperand *OperandBegin = MI.operands_begin(),
```

Comment:
Nit: Why not have `MachineOperand *OperandBegin = MI.operands_begin() + 1`, given that's it's actually the first operand we care about?